### PR TITLE
Upgrade duckdb to 0.9.0

### PR DESCRIPTION
**Pull request by alexeocto**
Newer versions of pandas are incompatible with previous versions of duckdb (I believe up to around 0.8.2), so upgrading duckdb version accordingly to avoid errors.

https://github.com/chroma-core/chroma/issues/1069

### CODE REVIEW

**Comment by josh-lynch** [comment]
Change looks fine, but can you split it up into separate commits so we can see what changes have come through in the commit history?

eg.

1. upgrade duckdb
2. bump version

If there's only a bump version commit it means we can't actually see what the changes are from the git log

**Comment by josh-lynch** [comment]
I think you have misunderstood my comment slightly, or possibly somethings just gone a little bit wrong with your git use. The initial commit should literally just be this one line:

```
"duckdb==0.9.0",
```

**Comment by alexeocto** [comment]
Sorry not sure what happened my end there, should be all correctly committed now 🤞

File: xocto/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "4.5.0"
+__version__ = "4.5.1"
```
Comment (pydanny) [inline xocto/__init__.py:1]: 1. Not a PATCH update - what bug does this fix?
2. Don't mix code changes with version updates

**Review by pydanny — CHANGES_REQUESTED** [review]
Unrelated version update in a version update

File: xocto/__init__.py (hunk 1 above)
Comment (alexeocto) [reply xocto/__init__.py:1]: If the user had a newer version of `pandas` then using the local filestore version of `select_object_contents` would error. Would that count as a bug?

File: setup.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
     zip_safe=False,
     install_requires=[
         "ddtrace>=1.9.0",
-        "duckdb==0.8.1",
+        "duckdb==0.9.0",
```
[the remark this answers is not in the document]
Comment (alexeocto) [reply setup.py]: Noted, have changed that thanks

File: xocto/__init__.py (hunk 1 above)
Comment (alexeocto) [reply xocto/__init__.py]: I've bumped this to 4.6.0 for now 👍

**Review by pydanny — APPROVED** [review]
Let's get this in to help our downstream friends. 😄

**Pull request merged**